homeassistant/components/tolosauna/climate.py

```diff
@@ -0,0 +1,168 @@
+"""TOLO Sauna climate controls (main sauna control)."""
+
+from typing import Any, Optional
```

Comment:
Add `from __future__ import annotations` and use `|` instead of `Optional`

Reply:
Done.